### PR TITLE
Email guessing

### PR DESCRIPTION
**Pull request by j6k4m8**
_Basic_ guessers for MSF and ACLU

### CODE REVIEW

**Comment by j6k4m8** [comment]
@dkleissa I branched from your dev-branch — feel free to merge in if you like (to your dev branch) and then we can merge it along with your changes into master when it's time.

**Comment by j6k4m8** [comment]
...well I don't _think_ that broke anything

File: api/charity.py [hunk 1]
```diff
@@ -20,4 +32,8 @@ def __init__(self):
         nickname='Get Supported Charities')
     def get(self):
         """Get Supported Charities"""
+        # Format the JSON response
```
Comment (slavingia) [inline api/charity.py]: these lines aren't needed. the front-end can handle the normal `return SUPPORTED_CHARITIES, 200`

Comment (slavingia) [reply api/charity.py]: meaning you can also remove the import of deepcopy i believe.

File: lambda/charity/charity.py [hunk 2]
```diff
@@ -17,24 +17,54 @@ def __init__(self, raw_email):
...
+
+        Arguments:
+            None
 
         Returns:
             (bool): True if it is the receipt for this charity
         """
-        raise NotImplemented
+        raise NotImplementedError
+
```
Comment (slavingia) [inline lambda/charity/charity.py:47]: one too many newlines

**Review by slavingia — APPROVED** [review]

**Pull request merged**